sdcm/cluster.py

```diff
@@ -242,7 +242,7 @@ class UserRemoteCredentials(object):
     def __init__(self, key_file):
         self.type = 'user'
         self.key_file = key_file
-        self.name = os.path.basename(self.key_file)[:-4]
+        self.name = os.path.basename(self.key_file)
```

Comment:
After this change we have 2 absolutely identical classes: UserRemoteCredentials and GCECredentials.
One of them should be removed.

Reply:
I agree with @larisau, however, I need this PR to be merged.
@bentsi, this commit is not related to this PR and I'd prefer you seperate it completely.
Please send a new PR to remove the duplicate class.